source/uwp/Renderer/idl/AdaptiveCards.Uwp.idl

```diff
@@ -1940,5 +1944,24 @@ namespace AdaptiveCards
             [default] interface IAdaptiveRenderArgs;
         };
 
+        [uuid(cced6652-8ca5-4841-b650-851770ec6ecf)]
+        [exclusiveto(WholeItemsPanel)]
+        [version(NTDDI_WIN10_RS1)]
+        interface IWholeItemsPanel : IInspectable
```

Comment:
>IWholeItemsPanel [](start = 18, length = 16)

Does this need to be exposed from our API?  It seems a little out of place in the AdaptiveCards.Uwp namespace...

Reply:
I'll see if it can be removed

---
In reply to: [148412119](https://github.com/Microsoft/AdaptiveCards/pull/866#discussion_r148412119) [](ancestors = 148412119)